instrumentation/opentelemetry_finch/lib/opentelemetry_finch.ex

```diff
@@ -67,6 +67,13 @@ defmodule OpentelemetryFinch do
         kind: :client
       })
 
+    if status >= 500 && status < 600 do
+      OpenTelemetry.Span.set_status(
+        s,
+        OpenTelemetry.status(:error, :opentelemetry_instrumentation_http.code_to_phrase(status))
```

Comment:
The reason cannot be set per convention. You can split the code to phrase to a separate PR or vice versa

Reply:
Oh yeah I see
> Don’t set the span status description if the reason can be inferred from http.response.status_code.

That should remove the need for the util stuff altogether. 👍  I'll remove it